### PR TITLE
Fix: profile sidebar width issue (#4156)

### LINKED ISSUE #4156: Profile sidebar width issue when leaving a reference
Subject: Profile sidebar width issue when leaving a reference
Description:
When leaving a reference for another user, the size of the profile sidebar on the left changes depending on the length of the user's location text.

It's a different size to the profile sidebar on a user's profile page.

Results:
It should be the same size regardless of contents.

Backend version: develop-3cdcce6b
Frontend version: develop-3cdcce6b
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:95.0) Gecko/20100101 Firefox/95.0
Page: https://app.couchers.org/
User: aapeli (1)

### PR DESCRIPTION
**Pull request by TiaApril**
Fixing the profile sidebar width issue, making it responsive when the city gets longer or shorter

Closes #4156  

**Web frontend checklist**
- 

- [ x ] Formatted my code with `yarn format`
- [ x ] There are no warnings from `yarn lint --fix`
- [ x ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ x ] Clicked around my changes running locally and it works
- [ x ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Comment by nabramow** [comment]
Hey @TiaApril if you run `yarn format` it should fix the prettier error that's failing the tests!

Also just FYI usually we fill out the frontend checklist in the PR description to go through the steps to show it's ready.

**Comment by TiaApril** [comment]
@nabramow sorry if I ask a lot, what should I do from here? I did run the yarn format from VScode, should I push the branch and do all over it again?

File: app/web/features/profile/view/leaveReference/LeaveReferencePage.tsx [hunk 1]
```diff
@@ -27,9 +27,12 @@ const useStyles = makeStyles((theme) => ({
...
+    padding: theme.spacing(1),
     [theme.breakpoints.up("md")]: {
-      display: "flex",
-      justifyContent: "space-around",
+      display: "grid",
+      gridTemplateColumns: "2fr 3fr",
+      gap: theme.spacing(3),
+      margin: theme.spacing(0, 3),
     },
   },
```
Comment (nabramow) [inline app/web/features/profile/view/leaveReference/LeaveReferencePage.tsx]: So this works at making the sidebar a stable width. However, let's make it the same width as the regular (non-reference) sidebar. That way when you click between `LeaveReference` and the regular profile the sidebar width is consistent.

You should be able to just copy the style from the `ProfilePage`:

app/web/features/profile/view/ProfilePage.tsx

Look at the style for `root`. I think you can just copy that root style into the `LeaveReference` root style to make them the same.

**Review by nabramow — APPROVED** [review]
Thanks for making that change! Looks great now, go ahead and merge it!

**Pull request merged**